source/agora/test/GossipProtocol.d

```diff
@@ -44,20 +44,19 @@ unittest
     nodes.each!(node => assert(node.getBlockHeight() == 0));
 
     Transaction[] last_txs;
-    foreach (block_idx; 0 .. 10)  // create 10 blocks
-    {
-        // create enough tx's for a single block
-        auto txs = makeChainedTransactions(getGenesisKeyPair(), last_txs, 1);
-
-        // send it to one node
-        txs.each!(tx => node_1.putTransaction(tx));
-
-        // gossip was complete
-        nodes.each!(node =>
-            txs.each!(tx =>
-                node.hasTransactionHash(hashFull(tx)).retryFor(1.seconds)
-        ));
-
-        last_txs = txs;
-    }
+    // create enough tx's for a single block
```

Comment:
Could you change the unittest documentation comment and write `/// test hasTransactionHash()`? Currently it's empty.

Reply:
Ah actually ignore my comment! I didn't realize this was a networking test.

Reply:
Okay :)